caffe2/CMakeLists.txt

```diff
@@ -628,37 +628,22 @@ if (BUILD_PYTHON AND BUILD_CAFFE2_OPS)
   endif()
 
   # Finally, Copy all python files to build directory
-  # Generate and create all needed __init__.py files, if they aren't already
-  # present in the current source tree.
-  message(STATUS "Automatically generating missing __init__.py files.")
-  caffe_autogen_init_py_files()
-
   # Create a custom target that copies all python files.
   file(GLOB_RECURSE PYTHON_SRCS RELATIVE ${PROJECT_SOURCE_DIR}
        "${PROJECT_SOURCE_DIR}/caffe2/*.py")
-  add_custom_target(python_copy_files ALL)
-  if(MSVC OR CMAKE_GENERATOR MATCHES "Ninja")
```

Comment:
Can we keep the Ninja check that chunks the copies? Is this no longer an issue?

Reply:
I have not seen that error repro and I am using ninja locally. Note the old code kept adding commands to a single target, while these adds multiple build rules and makes the target depend on them. My guess is that this comment is no longer relevant.